pro/src/pages/IndividualOffer/IndividualOfferDetails/components/IndividualOfferDetailsScreenNext.tsx

```diff
@@ -236,15 +237,14 @@ export const IndividualOfferDetailsScreenNext = ({
       images,
     } = product
 
-    const subCategory = subCategories.find(
-      (subCategory) => subCategory.id === subcategoryId
-    )
-
-    if (!subCategory) {
-      throw new Error('Unknown or missing subcategoryId')
+    const subcategory = subCategories.find((s) => s.id === subcategoryId)
+    if (!subcategory) {
+      return handleUnexpectedError(
+        new FrontendError('Unknown or missing `subcategoryId`.')
+      )
```

Comment:
Ça ne peut pas être changé en ça ?
```suggestion
    const subcategory = subCategories.find((s) => s.id === subcategoryId)
    assertOrFrontendError(subcategory, 'Unknown or missing `subcategoryId`.')
```